.github/workflows/lint_python.yml

```diff
@@ -6,16 +6,31 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v3
-      - run: pip install --upgrade pip wheel
-      - run: pip install black flake8 flake8-2020 flake8-bugbear
-                         flake8-comprehensions mypy pytest pyupgrade safety
-      - run: black --check . || true
-      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-      - run: flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88
-                      --show-source --statistics
-      - run: pip install -r pex-requirements.txt -r tests/requirements.txt
-      - run: mkdir --parents --verbose .mypy_cache
-      - run: mypy --ignore-missing-imports --install-types --non-interactive . || true
-      - run: pytest . || true
-      - run: pytest --doctest-modules . || true
-      - run: safety check
+        with:
+          cache: 'pip'
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip wheel
+          pip install .[all]
```

Comment:
I am not a fan of hiding what is being installed in some separate configuration file that is stored in some other directory.

Reply:
I would like to remove the requirements.txt and describe each dependency in extra dependencies.
In this way, when adding new dependencies, there is no need to rewrite multiple locations.

Reply:
OK.  If we go that way then let’s also put the sprawling flake8 settings in `setup.cfg`.

Reply:
Is the current setup.cfg setting not sufficient?
https://github.com/eggplants/internetarchive/blob/a18aea837bcaedbbd8d99417c0d9ff2f5bf1bea7/setup.cfg#L46-L74

Reply:
When pip's cache is working, it is not a problem to install using `all` extras each time.

Reply:
Getting there but missing the failing flake8-bugbear and flake8-comprehensions codes.

Reply:
>missing the failing flake8-bugbear and flake8-comprehensions codes

What is this?